src/Akeneo/Platform/Bundle/UIBundle/Resources/public/templates/form/creation/job.html

```diff
@@ -1,6 +1,6 @@
 <div class="AknFieldContainer">
     <div class="AknFieldContainer-header">
-        <label for="job" class="AknFieldContainer-label control-label required">
+        <label for="job" class="AknFieldContainer-label control-label required" title="Job">
```

Comment:
```suggestion
        <label for="job" class="AknFieldContainer-label control-label required">
```
This was for easy testing during our tests, it can be discarded